src/utilities/__tests__/buildClientSchema-test.js

```diff
@@ -341,21 +341,12 @@ describe('Type System: build schema from introspection', () => {
           value: 1,
         },
         FRUITS: {
-          description: 'Foods that are fruits.',
           value: 2,
         },
         OILS: {
-          description: 'Foods that are oils.',
+          deprecationReason: 'Too fatty',
```

Comment:
nit-pick: Can you put 'deprecationReason' after value to match the order in output?